pygorithm/fibonacci/space_optimized.py

```diff
@@ -0,0 +1,44 @@
+# Author: Ashutosh Gupta
+# Created On: 8/15/2017
+# Time: 3:24 PM
+
+"""
+nth fibonacci number - Space Optimized
+"""
+
+import inspect
+
+
+def get_sequence(n):
+    """
+    Return Fibonacci sequence from zero to specified number as list.
+    """
+
```

Comment:
Go ahead and follow one style — no space after function (or method in class) documentation.